doc/proposals/ansible-helm-addapi.md

```diff
@@ -0,0 +1,40 @@
+# Ansible/Helm add api command  Proposal for Operator SDK
+
+> Status: implementable
+> 
+
+- [Background](#background)
+- [Goals](#goals)
+- [Commands](#commands)
+- [Ansible_Changes] (#ansible changes)
+- [Helm_Changes] (#helm changes)
+
+## Background
+
+In current scenario, users are not able to add additional APIs for Ansible/Helm based opertators using SDK CLI.
+
+## Goals
+
+The goal of the proposal if implemented, is to enable users add additional APIs once the original Ansible/Helm operator is scaffolded.
+
+## Commands
+We are updating existing commands to accommodate the ansible/helm operator.  Changes to the `cmd\add\api.go` is needed.
+
+`operator-sdk add api --kind <kind> --api-version <group/version>`     
+
+## Ansible Changes
+
```

Comment:
For each new Kind, I understand that a dir is created so should we not create the new dir with the full structure? 

**Example**

1. We created the project with the Memcached API

── roles
│   └── Memcached
│       ├── README.md
│       ├── defaults
│       │   └── main.yml
│       ├── handlers
│       │   └── main.yml
│       ├── meta
│       │   └── main.yml
│       ├── tasks
│       │   └── main.yml
│       ├── tests
│       │   ├── inventory
│       │   └── test.yml
│       └── vars
│           └── main.yml

2. Then, the new Kind AppServer will be added:

├── roles
│   ├── AppServer
│   │   ├── README.md
│   │   ├── defaults
│   │   │   └── main.yml
│   │   ├── handlers
│   │   │   └── main.yml
│   │   ├── meta
│   │   │   └── main.yml
│   │   ├── tasks
│   │   │   └── main.yml
│   │   ├── tests
│   │   │   ├── inventory
│   │   │   └── test.yml
│   │   └── vars
│   │       └── main.yml
│   └── Memcached
│       ├── README.md
│       ├── defaults
│       │   └── main.yml
│       ├── handlers
│       │   └── main.yml
│       ├── meta
│       │   └── main.yml
│       ├── tasks
│       │   └── main.yml
│       ├── tests
│       │   ├── inventory
│       │   └── test.yml
│       └── vars
│           └── main.yml

Also, I understand that watches need to be updated as:

$ cat watches.yaml 
---
- version: v1alpha1
  group: cache.example.com
  kind: Memcached
  role: Memcached

- version: v1alpha1
  group: cache.example.com
  kind: AppServer
  role: AppServer

Reply:
Yes, Directory under "roles" is needed, looks like this point got missed in the proposed changes.  watches.yaml changes are noted already in the proposed changes

Reply:
Regarding the molecule directory, @fabianvf WDYT?

Reply:
I think we can close this one. I made a comment for him in the no-goals and shows that you addressed all already. 👍